go.work

```diff
@@ -0,0 +1,6 @@
+go 1.18
```

Comment:
Do we really need sub-modules? Can't we avoid the additional go workspace and have a single module with the e2e package in it?